### ISSUE TITLE
Centralize Logging Infrastructure

### ISSUE THREAD

**Issue by pxpeterxu** [coursetable/coursetable#9]
In GitLab by @hsheth2 on Sep 5, 2019, 20:36

Currently we use a home-grown PHP logging framework. It would be nice to transition to a standard and well-supported logging library instead.

Once this transition is done, it might enable us to use a better, more centralized logging mechanism and set up alerting for issues.

**Comment by hsheth2** [comment]
Based on my cursory research, it appears that [monolog](https://github.com/Seldaek/monolog) would be the best option.

**Comment by hsheth2** [comment]
Wontfix since this will be less relevant for the redesigned site.
